x-pack/test/apm_api_integration/basic/tests/metrics_charts/metrics_charts.ts

```diff
@@ -0,0 +1,441 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+import { first } from 'lodash';
+import { GenericMetricsChart } from '../../../../../plugins/apm/server/lib/metrics/transform_metrics_chart';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+import { expectSnapshot } from '../../../common/match_snapshot';
+
+interface ChartResponse {
+  body: {
+    charts: GenericMetricsChart[];
+  };
```

Comment:
Now you are at it we might as well fix it here as well :)

```suggestion
  body: MetricsChartsByAgentAPIResponse
```

from https://github.com/elastic/kibana/blob/e3f3d5fbf900fd2348818abaf85e0a76d00b703f/x-pack/plugins/apm/server/lib/metrics/get_metrics_chart_data_by_agent.ts#L15-L17